routes/click.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const utils = require('../helpers/utils');
+const router = express.Router();
+
+
+async function action(page, request) {
+    return Promise.all([
```

Comment:
- I think you should await for promises and form regular response
- we should consider waiting for navigation

Reply:
The pattern for doing this was brought from [pupeteer documentation](https://github.com/GoogleChrome/puppeteer/blob/v1.19.0/docs/api.md#pageclickselector-options).

Reply:
The pattern you are referring to uses waitForNavigation. You are using waitFor. Their promises resolve to different objects.
Nonetheless, I think this action should capture page state after click regardless of waiting method.